blocks/api/source.js

```diff
@@ -30,6 +25,10 @@ function withKnownSourceFlag( fn ) {
 	} );
 }
 
+function toArray( type, props, ...children ) {
+	return [ [ type, omit( props, 'key' ), ...children ] ];
```

Comment:
When will props have `key` ?

Reply:
That's from Dom-react... As mentioned above, maybe it's better that we remove this dependency?